modules/ding_nodelist/ding_nodelist.module

```diff
@@ -0,0 +1,2477 @@
+<?php
+/**
+ * @file
+ * Helps creating customizable lists of content that can be placed on a panel.
+ */
+
+include_once 'ding_nodelist.features.inc';
+
+define('NL_WIDGET_LIST', 'simple');
+define('NL_WIDGET_SINGLE', 'single');
+define('NL_WIDGET_CAROUSEL', 'carousel');
+define('NL_WIDGET_HOR_ACCORDION', 'horizontal_accordion');
+define('NL_WIDGET_VER_ACCORDION', 'vertical_accordion');
+define('NL_WIDGET_SLIDER', 'slider');
+define('NL_WIDGET_TAXONOMY_LIKE', 'taxonomy');
+
+define('NL_TPL_ACTIVE', 0);
+define('NL_TPL_HIDDEN', 1);
+
+define('NL_CACHE_LIFETIME', 1800);
+define('NL_TITLE_SIZE', 25);
+define('NL_BODY_SIZE', 100);
```

Comment:
Please prefix constants with the actual module name. Here: `DING_NODELIST`. I know it becomes longer to type but may IDEs will help here anyway.